boilerplate/test/test-base-convention/includes.mk

```diff
@@ -1,4 +0,0 @@
-# This is a rollup of includes in this convention, for easier inclusion
-# by the repo's Makefile, which should just include this file.
-include boilerplate/openshift/golang_osd_cluster_operator/project.mk
-include boilerplate/openshift/golang_osd_cluster_operator/standard.mk
```

Comment:
These bogus includes were killing the test case when it needed to actually use `make` in this convention.